frontend/src/component/PublicProfile/ProfileMainBar.jsx

```diff
@@ -1,27 +1,45 @@
 import React from 'react'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+
 import {
   Avatar,
   AVATAR_SIZE,
-  Breadcrumbs
+  Breadcrumbs,
+  IconButton
 } from 'tracim_frontend_lib'
-import PropTypes from 'prop-types'
 
-export const ProfileMainBar = props => {
-  const hasUser = Object.keys(props.displayedUser).length !== 0
+import { FETCH_CONFIG } from '../../util/helper.js'
 
+export const ProfileMainBar = props => {
+  const hasUser = Object.keys(props.displayedUser).length > 0
   return (
     <div className='profile__mainBar'>
-      <Avatar
-        customClass='profile__mainBar__bigAvatar'
-        publicName={hasUser ? props.displayedUser.publicName : ''}
-        size={AVATAR_SIZE.BIG}
-        style={{ position: 'relative', top: '-15px' }}
-      />
-      <Avatar
-        customClass='profile__mainBar__mediumAvatar'
-        publicName={hasUser ? props.displayedUser.publicName : ''}
-        size={AVATAR_SIZE.MEDIUM}
-      />
+      <div className='profile__mainBar__avatar' data-cy='profile_avatar'>
+        <Avatar
+          customClass='profile__mainBar__avatar__big'
+          apiUrl={FETCH_CONFIG.apiUrl}
+          user={hasUser ? props.displayedUser : {}}
+          size={AVATAR_SIZE.BIG}
+          filenameInUrl={props.avatarFilenameInUrl}
+        />
+        <Avatar
+          customClass='profile__mainBar__avatar__medium'
+          user={hasUser ? props.displayedUser : {}}
+          size={AVATAR_SIZE.MEDIUM}
+          filenameInUrl={props.avatarFilenameInUrl}
+        />
+        {props.changeAvatarEnabled && (
+          <IconButton
+            text='' title={props.t('Change avatar')}
```

Comment:
```suggestion
            text='' 
            title={props.t('Change avatar')}
```

`text` is required?

Reply:
It was but isn't anymore, I'll remove it.